README.md

```diff
@@ -22,22 +22,61 @@ mkcert -cert-file tls-cert.pem -key-file tls-key.pem localhost 127.0.0.1 ::1
 
 One caveat is HTTP redirect to HTTPS is not supported, you have to type in https in the browser address bar manually.
 
+## Prerequisite
+
+Note that there is a local .tool-version in project root. For the following setup to work, we need to
+
+1. Install asdf
+
+2. Run (please refer to the version in .tool-version)
+   ```sh
+   asdf install nodejs 12.18.3
+   asdf install golang 1.15
+   ```
+
 ## Database setup
 
 1. Setup dependencies:
    ```sh
    make vendor
    ```
-2. Create a schema:
-   ```sql
-   CREATE SCHEMA app;
-   ```
-3. Setup environment variables (in `.env`):
+2. Setup environment variables (in `.env`):
+
+   copy from `.env.example`, then add
```

Comment:
Sorry this part is outdated, we don't need the two DATABASE_ environment variables anymore. Can you help removing them?